packages/flutter/lib/src/widgets/text.dart

```diff
@@ -781,6 +803,13 @@ class Text extends StatelessWidget {
         child: ExcludeSemantics(excluding: semanticsLabel != null, child: result),
       );
     }
+    final double? paragraphSpacing = MediaQuery.maybeParagraphSpacingOf(context);
```

Comment:
For paragraph spacing, what do you think of leaving a TODO comment to add support for spacing paragraphs within a `Text` widget, with a link to an issue? Same comment for `SelectableText`.

Reply:
Sounds good to me.